package.json

```diff
@@ -18,7 +18,7 @@
     "deploy": "mv dist/index.html functions/ && firebase deploy --only functions,hosting",
     "e2e:serve_and_retest_on_change": "xvfb-maybe start-server-and-test serve:dev http://localhost:4200 'npm run e2e:retest_on_change'",
     "e2e:retest_on_change": "xvfb-maybe nodemon -e js,jsx,ts,tsx,css,scss --exec 'ng test -- --watch=false; cypress run --headed --no-exit'",
-    "e2e:autorun": "xvfb-maybe ng test -- --watch=false; xvfb-maybe cypress run --headed --no-exit",
```

Comment:
tests don't seem to run with "--no-exit". They just stop and wait after one Cypress suite. This change seems to achieve the desired effect.